monai/bundle/scripts.py

```diff
@@ -507,6 +529,9 @@ def run(
             if it is a list of file paths, the content of them will be merged.
         logging_file: config file for `logging` module in the program, default to `None`. for more details:
             https://docs.python.org/3/library/logging.config.html#logging.config.fileConfig.
+        tracking: if "mlflow", will add `MLFlowHandler` to the parsed bundle with default loggging settings,
```

Comment:
Maybe explain here that this is the point of enabling experiment managements, optionally configurable and extensible... We may need a link to a tutorial here

Reply:
Sure, enhanced the doc-string and added an example config, will also add the tutorial in future when it's ready.

Thanks.